docs/gestor_dependencias.md

```diff
@@ -0,0 +1,59 @@
+# Gestor de Dependencias
+
+La gestión de dependencias es esencial para asegurar que el proyecto pueda integrarse con bibliotecas externas de manera eficiente, manteniendo su estabilidad y facilidad de mantenimiento. Este documento detalla los gestores de dependencias considerados para el proyecto y las razones detrás de su selección. Se han considerado tres opciones: Apache Maven, Gradle y Apache Ivy.
+
+## Criterios de Selección
+
+Para elegir uno de los gestores, nos basaremos en una serie de criterios:
+
+- *Eficiencia*: Tiempo medio para resolver dependencias en un proyecto estándar (ms).
+- *Comunidad*: Cantidad de recursos y paquetes desarrollados, soporte en foros y documentación oficial actualizada.
+- *Compatibilidad*: Porcentaje de frameworks y bibliotecas Java populares soportados de forma nativa.
+
+---
+
+## Gestores de Dependencias Evaluados
+
+### 1. Apache Maven
+
+- **Descripción**: Maven es una herramienta de gestión de proyectos y dependencias ampliamente utilizada en el ecosistema Java. Utiliza un archivo `pom.xml` como base para definir configuraciones y dependencias.
```

Comment:
¿Aporta algo a la justificación de la elección que use un fichero u otro? ¿Es un fichero especificado en algún estándar?
Por otro lado, *siempre* es conveniente que pongáis un enlace a la web de la herramienta.